packages/fetchai/connections/p2p_libp2p/libp2p_node.go

```diff
@@ -69,6 +65,20 @@ func check(err error) {
 	}
 }
 
+func ignore(err error) {
+	if err != nil {
+		log.Println("TRACE", err)
+	}
+}
+
+/*
+// DHTNode libp2p node interface
+type DHTNode interface {
+	RouteEnvelope(aea.Envelope) error
+	ProcessEnvelope(func(aea.Envelope) error)
+}
+*/
+
```

Comment:
Remove?

Reply:
This will be the interface that libp2p_node uses to route and receive envelope to/from the dht as an abstraction of DHTPeer and DHTClient implementations